tests/e2e/General/UsageTest.php

```diff
@@ -0,0 +1,599 @@
+<?php
+
+namespace Tests\E2E\General;
+
+use Tests\E2E\Client;
+use Tests\E2E\Scopes\ProjectCustom;
+use Tests\E2E\Scopes\Scope;
+use Tests\E2E\Scopes\SideServer;
+use CURLFile;
+use Tests\E2E\Services\Functions\FunctionsBase;
+
+class UsageTest extends Scope
+{
+    use ProjectCustom;
+    use SideServer;
+    use FunctionsBase;
+
+    protected array $headers = [];
+    protected string $projectId;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+    }
+
+    public function testUsersStats(): array
+    {
+        $project = $this->getProject(true);
+        $projectId = $project['$id'];
+        $headers['x-appwrite-project'] = $project['$id'];
+        $headers['x-appwrite-key'] = $project['apiKey'];
+        $headers['content-type'] = 'application/json';
+
+        $usersCount = 0;
+        $requestsCount = 0;
+        for ($i = 0; $i < 10; $i++) {
+            $email = uniqid() . 'user@usage.test';
+            $password = 'password';
+            $name = uniqid() . 'User';
+            $res = $this->client->call(Client::METHOD_POST, '/users', $headers, [
+                'userId' => 'unique()',
+                'email' => $email,
+                'password' => $password,
+                'name' => $name,
+            ]);
+            $this->assertEquals($email, $res['body']['email']);
+            $this->assertNotEmpty($res['body']['$id']);
+            $usersCount++;
+            $requestsCount++;
+
+            if ($i < 5) {
+                $userId = $res['body']['$id'];
+                $res = $this->client->call(Client::METHOD_GET, '/users/' . $userId, $headers);
+                $this->assertEquals($userId, $res['body']['$id']);
+                $res = $this->client->call(Client::METHOD_DELETE, '/users/' . $userId, $headers);
+                $this->assertEmpty($res['body']);
+                $requestsCount += 2;
+                $usersCount--;
+            }
+        }
+
+
+        sleep(35);
```

Comment:
Are these sleeps here to wait for the usage task to trigger? If so, is it possible to manually trigger the task? Maybe with `Console::exec()`?

Reply:
We can manually trigger, but here being e2e test, we want to test the usage container does the work timely and keeps updating the stats. Also there is the part stated aggregating data to influxdb I think that happens every 10s, we cannot manually trigger that.